lib/matplotlib/_mathtext.py

```diff
@@ -1481,7 +1562,8 @@ def hlist_out(box):
                     rule_depth = box.depth
                 if rule_height > 0 and rule_width > 0:
                     cur_v = base_line + rule_depth
-                    p.render(cur_h + off_h, cur_v + off_v,
+                    p.render(output,
```

Comment:
Same here, if a test can hit this, it would be great!

Reply:
This line is already covered -- try adding a print and run e.g. `figtext(.5, .5, r"\frac{1}{2}")` (and there's definitely tests for fractions).  Likely codecov is just being flaky here.